libcasr/src/asan.rs

```diff
@@ -231,6 +234,113 @@ impl Severity for AsanContext {
     }
 }
 
+/// Structure provides an interface for save parsing some sanitizer crash.
+#[derive(Clone, Debug)]
+pub struct SanCrash {
+    context: StacktraceContext,
```

Comment:
this is just a duplicate of StacktraceContext, do we need a new structure here? Or we can just create an alias?

Reply:
Yes, we need, cuz there's no structure inheritance in Rust and generics without any generic type parameter.